src/bindings/python/src/pyopenvino/graph/ops/paged_attention_extension.cpp

```diff
@@ -11,7 +11,7 @@
 namespace py = pybind11;
 
 void regclass_graph_op_PagedAttentionExtension(py::module m) {
-    using ov::op::PagedAttentionExtension;
+    using ov::op::internal::PagedAttentionExtension;
```

Comment:
Is the change adding "internal" namespace crucial for this PR?
I don't say it's not a good move, but it could be done separately.
Included within this PR increase the changes to review and changed files significantly (touching several different components).

Reply:
Oki, undid the changes